### PR TITLE
CodeArtifact Migration - Phase 1

### PR DESCRIPTION
**Pull request by EvanLemieux**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,16 +39,28 @@
...
+    "eslint-plugin-html": "^6.2.0",
+    "eslint-plugin-lit": "^1.6.1",
+    "eslint-plugin-sort-class-members": "^1.14.1",
     "polymer-cli": "^1.9.1"
+  },
+  "files": [
+    "fullscreen-api.js",
+    "fullscreen-behavior.js",
+    "fullscreen-icon-button.js",
+    "fullscreen-icons.js"
```
Comment (dlockhart) [inline package.json:61]: Not sure what `elements.html` is doing, but I'm guessing it was a pre-Polymer 3 artifact that isn't needed anymore.

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+14
```
[the remark this answers is not in the document]
Comment (EvanLemieux) [reply .nvmrc:1]: I didn't try, I'll update it!

File: package.json (hunk 1 above)
Comment (EvanLemieux) [reply package.json:61]: I will delete it, I don't see any references.

File: package.json [hunk 3]
```diff
@@ -18,14 +18,11 @@
     "url": "https://github.com/Brightspace/fullscreen-api.git"
   },
   "homepage": "https://github.com/Brightspace/fullscreen-api",
```
Comment (svanherk) [inline package.json:20]:
```suggestion
    "url": "https://github.com/BrightspaceUILabs/fullscreen-api.git"
  },
  "homepage": "https://github.com/BrightspaceUILabs/fullscreen-api",
```

**Review by svanherk — APPROVED** [review]
🙌 🎊 🥇

**Pull request merged**